Index.html

```diff
@@ -52,10 +52,182 @@ <h1>
+                <ul class="job-description">
+                    <li>CANOPY</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="work-technologie">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <button class="button">See Project</button>
+            </div>
+         </article>
+    
+         <article class="portfolio-work">
+            <img class="work-image" src="./assets/Snapshoot Portfolio 2.svg" alt="">
+        <div class="work-description">
+            <h1 class="work-title">Multi-Post <br>
+                Stories</h1>
+            <ul class="job-description">
+                <li>CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="work-technologie">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+            </ul>
+            <button>See Project</button>
+        </div>
+     </article>
+
+     <article class="portfolio-work">
+        <img class="work-image" src="./assets/Snapshoot Portfolio 3.svg "alt="">
+    <div class="work-description">
+        <h1 class="work-title">Tonic</h1>
+        <ul class="job-description">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+        </ul>
+        <p>Tonic</p>
+        <ul class="work-technologie">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+        </ul>
+       <div> 
+        <button>See Project</button>
+    </div>
+    </div>
+ </article>
+ <article class="portfolio-work">   
+    <img class="work-image" src="./assets/Snapshoot Portfolio 4.svg" alt="">
+<div class="work-description">
+    <h1 class="work-title">Multi-Post <br> Stories</h1>
+    <ul class="job-description">
+        <li>CANOPY</li>
+        <li>Back End Dev</li>
+        <li>2015</li>
+    </ul>
+    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="work-technologie">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+    </ul>
+    <button class="button-btn">See Project</button>
+</div>
+</article>
+</section>
+
+<section id="portfolio">
+    <article class="portfolio-work">
+        <div>
+            <h1>About <br> Myself</h1>
+            <p>Hello I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+         </div>
```

Comment:
- Kindy, stick your design to Figma, you designed it like other cards but it is different in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=C0vU075hnDfZLPwM-0).👌😔


| Figma design                                                             | Your  design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/206514549-72ae8697-f628-477a-a4e8-b7ffc9ce144a.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/206514558-fd99e578-0808-4ebf-925c-1bf3e06ce895.png" width="200">|


😎️